### DISCUSSION TITLE
google-cloud-pub-sub/google-cloud-pub-sub-grpc GA

### DISCUSSION THREAD

**Discussion by nicksabol** [apache/pekko#574] [Q&A] [answered]
Hi all, my company is planning a move to Pekko from Akka for both our Kafka and Pub/Sub implementations. There seem to be GA Pekko equivalents for our Kafka usage, but the Pub/Sub connector libraries are not GA yet. Do we have a timeline for when they might be? We are also open to helping take on work to get them there. Thank you very much

**Comment by pjfanning** [comment, marked as answer]
The Google Connectors are part of Pekko Connectors. Pekko Connectors will hopefully be released soon. It is currently blocked waiting for Pekko gRPC to be released (an RC of which is being voted on at the moment).

https://cwiki.apache.org/confluence/display/PEKKO/Pekko+Module+Release+Order

The best way to follow to releases is to subscribe to dev@pekko.apache.org mailing list.

https://lists.apache.org/list.html?dev@pekko.apache.org

**Reply by nicksabol** [reply]
Great news! Thanks very much. I'll follow along with the dev mailing list.

**Reply by nicksabol** [reply]
@pjfanning just to clarify for my team, when you said "Pekko Connectors will hopefully be released soon" did you mean all of Pekko connectors including plain Pub/Sub or just the gRPC Pub/Sub connector? Thanks

**Reply by pjfanning** [reply]
We don't release 1 jar at a time - please look at https://github.com/apache/incubator-pekko-connectors -- when I say 'release Pekko Connectors', I mean we release all the jars associated with that git repo.

This is exactly how Akka/Alpakka is released too.

**Reply by pjfanning** [reply]
The release discussion is at https://lists.apache.org/thread/gd4mhjf5zfd4gcdzcv92jsfy5hnv2ggg

If we don't get many responses on that mail thread, the default position will be to wait a few more weeks before starting the release process.

Pekko gRPC is released so the Google connectors are releasable but the main blocker now is the opinion (not mine) that we should wait on Scala 3 support for Slick before releasing any Pekko connectors.

**Reply by nicksabol** [reply]
I have sent an email response but don't see it on the site yet so I'll repost here to try to improve my chances of being heard before a decision is made. 

I do not personally see much value in waiting for Slick 3.5.0 and Scala 3.3.1 to be released before beginning the RC process. We are running out of time before the latest open source Akka version is no longer going to be maintained putting many people at security risk, and the Slick and Scala 3 requirement does not feel high priority. I would really like to see us begin the RC process to enable a prompt GA.
